src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -45,5 +49,5 @@
       {:style                             style/accounts-list
        :content-container-style           style/accounts-list-container
        :data                              accounts
-       :render-fn                         render-fn
+       :render-fn                         #(render-fn % network-details)
```

Comment:
Although this works, our standard is to pass extra data using the prop `:render-data`.

Reply:
Done